Task 1 complete

Ark82 opened the pull request:
کار های ساخت اکانت جدید تموم شد

Ark82 committed:
Task 1 complete

Ark82 commented:
کار های ساخت اکانت جدید تموم شد

Nima-ghaferi commented on Bank/src/entity/Customer.java:
```diff
@@ -7,10 +7,10 @@
 public class Customer {
     // fields
     private int Id;
+    private static List<Integer> IdList = new ArrayList<>();
```
in liste chie?

Ark82 replied:
این لیست برای این هست که اگر آی دی فردی که قراره صاحب اکانت میشه یک کاستومر واقعی باشه با آی دی واقعی به خاطر همین تمام آی دی های ثبت شده رو توی ی لیست ذخیره کردم تا بتونم از متود contains استفاده کنم و تو فایل AccountBiz خط 31 استفاده شده اگر طرز بهتری برای نوشتن اون قسمت هست بفرمایید تا تغییرش بدم.

Nima-ghaferi commented on Bank/src/entity/Customer.java:
```diff
@@ -104,6 +108,7 @@ public Customer(int Id, String firsName, String lastName, Date birthDate, String
         this.address = address;
         this.emailAddress = emailAddress;
         this.nationality = nationality;
+        IdList.add(Id);
```
اینم مثل بالایی نگرفتم که چیه

Nima-ghaferi commented on Bank/src/main/Main.java:
```diff
@@ -9,17 +9,25 @@ public static void main(String[] args) {
                 continue;
             }
+
+            if(choice.equalsIgnoreCase("2")) {
```
if -> else if

Nima-ghaferi commented on Bank/src/main/Main.java:
```diff
@@ -9,17 +9,25 @@ public static void main(String[] args) {
             System.out.println("4.");
             System.out.println("5.");
             System.out.println("6.");
             System.out.println("7. Quit");
```
7 -> q

Nima-ghaferi commented on Bank/src/main/Main.java:
```diff
@@ -9,17 +9,25 @@ public static void main(String[] args) {
+                continue;
+            }
+
             if(choice.equalsIgnoreCase("q"))
```
if -> else if

Ark82 committed:
Issues fixed (else if's)

Ark82 committed:
Issue's fixed

Added a validateId in AccountBiz and deleted customer id lists in Customer class.

Ark82 committed:
AccountBiz changes and comment removal

اضافه کردن آی دی به اکانت ها و ایجاد تغییرات در گرفتن شماره کارت و پاک کردن تمام کامنت ها

Ark82 committed:
Added findById methods and some other changes

متود اضافی برای پیدا کردن متغیر ها از کلاس اکانت و کاستومر از طریق آی دی ساخته شد
